types/ros2d/ros2d-tests.ts

```diff
@@ -0,0 +1,34 @@
+import {ROS2D} from './index'
```

Comment:
You shouldn't need to import this object since you're working with a global module (i.e., this should reflect what the user experience would be like).